cmd/support_archive/resources_test.go

```diff
@@ -189,7 +185,10 @@ func TestManifestCollector_NoManifestsAvailable(t *testing.T) {
 
 	ctx := context.TODO()
```

Comment:
IMHO context.Background() could be used directly in `newK8sObjectCollector(ctx...)` call. The variable is not reused .

Reply:
fixed in d485e00a